### ISSUE TITLE
output contains a new line

### ISSUE THREAD

**Issue by tsureshkumar** [susam/toytotp#8]
It would be good if new line is not present when copying the token to clipboard and later paste into other prompts.  The new line makes login fail as one extra character is present.

```
$ gpg -q -o - secret.gpg | mintotp | pbcopy
123456

```

**Comment by tsureshkumar** [comment]
Either this can be fixed in code to not to print newline with ```print(token, end='')``` or the README can be changed with trimming the new line when copying to clipboard. 

```
$ gpg -q -o - ~/.secrets/vpn-tf-auth.gpg | mintotp | tr -d '\n' | pbcopy
```

**Comment by susam** [comment]
Thank you for reporting this issue. Fixed it by updating the README in commit  e6d1466.
